IFComp/root/lib/site/header

```diff
@@ -65,7 +67,7 @@ END;
               </li>
           </ul>
           <form class="navbar-form navbar-right" role="form" action="[% form_action %]" method="POST">
-            <button type="submit" class="btn btn-success">[% button_label %]</button> 
+            <button type="submit" id="[% button_id %]" class="btn btn-success">[% button_label %]</button> 
```

Comment:
Curious where this change came from, but it's by no means a blocker (and it appears to assign a valid ID anyway)

Reply:
I believe it was added to allow the Selenium test to find the button by id.